imports/ui/accounts/Account.jsx

```diff
@@ -209,7 +211,12 @@ export default class AccountDetails extends Component {
                 else {
                     let totalValue = cloneDeep(this.state.total);
                     for (let v in totalValue) {
-                        totalValue[v].amount = parseFloat(totalValue[v].amount) + parseFloat(this.state.unbonding[v].amount)
+                        for (let c in this.state.unbonding) {
```

Comment:
Please describe the issue and how the issue was solved with this piece of codes.